Osteppy/src/server.js

```diff
@@ -1,32 +1,97 @@
 #!/usr/bin/env node
 
-/***********************************************************
+/** ********************************************************
 // OSTEP Dashboard Osteppy API
 // server.cpp
 // Date Created: 2018/09/11
 // Author: Olga Belavina and Yiran Zhu
 // Email: yzhu132@myseneca.ca
-// Description: Slack API triggered by and responds to 
-// slash commands such as /eod and /eod_left. 
-***********************************************************/
+// Description: Slack API triggered by and responds to
+// slash commands such as /eod and /eod_left.
+********************************************************* */
 
+import cp from 'child_process';
 import http from 'http';
+import path from 'path';
 import express from 'express';
 import cors from 'cors';
 import morgan from 'morgan';
 import bodyParser from 'body-parser';
-import config from './config.json';
 import axios from 'axios';
-import * as fs from 'fs';
+import fs from 'fs';
+import config from './config.json';
+
+class EODList {
```

Comment:
Would it make sense to split up the server & EODlist into 2 modules?

Reply:
Sure, can definitely do this 👍